src/models/Polygon.js

```diff
@@ -7,7 +7,18 @@ const options = {
     polygonSides: {
       title: 'Number of sides',
       min: 3
-    }
+    },
+    roundCorners: {
+      title: 'Round Corners',
+      type: 'checkbox',
+    },
+    roundFraction: {
+      title: 'Round Fraction',
+      min: 0.0,
+      max: 1.0,
```

Comment:
With a min of 0, you get no rounding and with a max of 1.0 you get a circle no matter how many polygon sides. I'm torn on whether it's good to be able to visualize these extremes, vs perhaps going with a min = 0.1, and max = 0.85 to reflect that "round corners" does what it advertises.

Reply:
You're definitely right. I also think more fine grained detail in the lower range is helpful, and the upper range isn't very interesting, so I changed it to 0.05, 0.5, with a step of 0.025. That looks better to me.